### PR TITLE
IGNITE-14585 IgniteSpec can provide arbitrary configs

### PR DESCRIPTION
**Pull request by nizhikov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

**Review by x-kreator — APPROVED** [review]

File: modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py [hunk 1]
```diff
@@ -207,14 +205,14 @@ def _prepare_config(self, node):
 
         config.discovery_spi.prepare_on_start(cluster=self)
 
-        node_config = self.spec.config_template.render(config_dir=self.config_dir, work_dir=self.work_dir,
-                                                       config=config)
+        for template in self.spec.config_templates:
```
Comment (anton-vinogradov) [inline modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py:208]: Use named  params instead, eg. "name_x, name_y"

**Review by anton-vinogradov — CHANGES_REQUESTED** [review]

File: modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py [hunk 2]
```diff
@@ -193,9 +193,7 @@ def init_persistent(self, node):
         """
         super().init_persistent(node)
 
-        node_config = self._prepare_config(node)
-
-        node.account.create_file(self.config_file, node_config)
+        self._prepare_config(node)
 
     def _prepare_config(self, node):
```
Comment (anton-vinogradov) [inline modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py:198]: should this be _prepare_configs now?

File: modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py (hunk 1 above)
Comment (nizhikov) [reply modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py]: Good catch. Thanks, fixed.

File: modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py (hunk 2 above)
Comment (nizhikov) [reply modules/ducktests/tests/ignitetest/services/utils/ignite_aware.py:198]: Fixe.

**Review by anton-vinogradov — APPROVED** [review]

**Pull request merged**